### PR TITLE
Added Trufflehog

### PR DESCRIPTION
**Pull request by GPelayo**
The addition to the gitlab script is only a stand-in until testing architecture is complete.

### CODE REVIEW

File: .gitlab-ci.lower-env.yml [hunk 1]
```diff
@@ -24,6 +24,17 @@ cache:
         - public
         - maintenance
 
+
+# Test with Trufflehog
+test_trufflehog:
+  stage: build
+  image: python:3.5
+  before_script:
+    - echo pip install trufflehog
```
Comment (hannes-ucsc) [inline .gitlab-ci.lower-env.yml]:
```suggestion
    - pip install trufflehog
```

File: .gitlab-ci.lower-env.yml [hunk 2]
```diff
@@ -24,6 +24,17 @@ cache:
...
+  script:
+    - echo trufflehog --regex --rules trufflehog-rules.json --branch develop --entropy=False --max_depth 30 https://github.com/HumanCellAtlas/data-portal.git
```
Comment (hannes-ucsc) [inline .gitlab-ci.lower-env.yml]:
```suggestion
    - trufflehog --regex --rules trufflehog-rules.json --branch develop --entropy=False --max_depth 30 https://github.com/HumanCellAtlas/data-portal.git
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
And just to confirm, you mentioned that Trufflehog actually found a secret in the code base? Why isn't that being detected in https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/data-portal/-/jobs/29602 where the `echo` was absent?

**Comment by GPelayo** [comment]
~~I probably meant the data-browser codebase. The data-portal didn't contain any secrets.~~ The secret I found was in the data-browser codebase and it popped up before I added the argument `--max_depth 30` to the Trufflehog command. I added this arg because Azul only checks secrets in the 30 most recent commits and I want to keep the Trufflehog configurations consistent.

**Comment by hannes-ucsc** [comment]
@GPelayo, please Slack (!) me the commit and location of that secret.

**Review by hannes-ucsc — APPROVED** [review]

**Pull request merged**